assets/source/setup-guide/app/style.scss

```diff
@@ -508,13 +508,22 @@ $root: ".woocommerce-setup-guide";
 					}
 
 					img {
+						padding: 12px;
 						height: 100%;
 						object-fit: cover;
 						width: 100%;
 					}
 				}
 			}
 		}
+
+		&__tac-modal {
+			width: 600px;
+
+			.components-modal__header {
+				border-bottom: 1px solid #ccc;
```

Comment:
Minor comment, the default border color is #ddd, we could keep the default instead of adding an extra CSS rule for this.